bmaptools/BmapCopy.py

```diff
@@ -211,6 +213,20 @@ def __init__(self, image, dest, bmap=None, image_size=None):
 
         self._batch_blocks = self._batch_bytes // self.block_size
 
+    def set_psplash_pipe(self, pipe):
+        """
+        Set the psplash named pipe file path to be used when updating the
+        progress.
+
+        The 'pipe' argument is the named pipe used by the psplash process to get
+        progress bar commands.
+        """
```

Comment:
I think the convention n this code is to have a newline after the docstring, lets keep it consistent.

Reply:
Thanks. I'll address it.